textile/features.textile

```diff
@@ -203,7 +203,7 @@ h3(#rest-auth). Auth
 * @(RSA6)@ The @capability@ for new tokens is JSON stringified. If If the user-provided @tokenParams@ does not specify capabilities, the @capability@ field should be omitted from the @tokenRequest@, and Ably will supply a token with the capabilities of the underlying key. See "TK2b":#TK2b
 * @(RSA7)@ @clientId@ and authenticated clients:
 ** @(RSA7d)@ If @clientId@ is provided in @ClientOptions@ and @RSA4@ indicates that token auth is to be used, the @clientId@ field in the @TokenParams@ (@TK2c@) should be set to that @clientId@ when requesting a token
-** @(RSA7e)@ If @clientId@ is provided in @ClientOptions@ and @RSA4@ indicates that basic auth is to be used, then:
+** @(RSA7e)@ If a valid @clientId@ is provided in @ClientOptions@, then:
```

Comment:
Wdyt about
```suggestion
** @(RSA7e)@ If a non-wildcard @clientId@ is provided in @ClientOptions@, then:
```

Reply:
I don't like that for DRY reasons. RSA7c defines what a valid clientId is, if we add additional restrictions that should be the one place we need to edit.

Reply:
Okay, makes sense.